scout/parse/variant/genotype.py

```diff
@@ -411,21 +420,77 @@ def _parse_format_entry(variant, pos, format_entry_name):
     alt = None
     if format_entry_name in variant.FORMAT:
         try:
-            value = variant.format(format_entry_name)[pos]
-            values = list(value.split("/"))
+            values = variant.format(format_entry_name)[pos]
+
+            new_values = []
+            for value in values:
+                for delim in ["/", ","]:
+                    if delim in value:
+                        new_values = list(value.split(delim))
+            if new_values:
+                values = new_values
 
             ref_value = None
             alt_value = None
 
             if len(values) > 1:
-                ref_value = int(values[0])
-                alt_value = int(values[1])
+                ref_value = (number_format)(values[0])
+                alt_value = (number_format)(values[1])
             if len(values) == 1:
-                alt_value = int(values[0])
+                alt_value = (number_format)(values[0])
             if ref_value >= 0:
                 ref = ref_value
             if alt_value >= 0:
                 alt = alt_value
         except (ValueError, TypeError) as _ignore_error:
             pass
     return (ref, alt)
+
+
+def _parse_format_entry_trgt_mc(variant, pos):
+    """Parse genotype entry for TRGT FORMAT MC
+
+    Args:
+        variant:
+        pos:
+
+    Returns:
+        mc_ref, mc_alt
+    """
+
+    mc_ref = None
+    mc_alt = None
+    pathologic_struc = variant.INFO.get("PathologicStruc", None)
+
+    pathologic_counts = 0
```

Comment:
```suggestion
    pathologic_struc = variant.INFO.get("PathologicStruc", None)

    pathologic_counts = 0
```

These 2 could be moved down, after line 468 perhaps?